core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -71,7 +71,7 @@ class KafkaApis(val requestChannel: RequestChannel,
   def handle(request: RequestChannel.Request) {
     try {
       trace("Handling request:%s from connection %s;securityProtocol:%s,principal:%s".
-        format(request.requestObj, request.connectionId, request.securityProtocol, request.session.principal))
+        format(request.requestDesc(false), request.connectionId, request.securityProtocol, request.session.principal))
```

Comment:
Thanks for updating. Is it correct to pass `false` here? I had a look at a few of the `toString` implementations and they all seemed to call `describe(true)`. Have you seen something different?